ipatests/test_integration/test_installation.py

```diff
@@ -772,3 +773,49 @@ def test_files_ownership_and_permission_teardown(self):
         """ Method to restore the default bashrc contents"""
         if self.bashrc_file is not None:
             self.master.put_file_contents('/root/.bashrc', self.bashrc_file)
+
+
+class TestInstallMasterReplica(IntegrationTest):
+    """https://pagure.io/freeipa/issue/7929
```

Comment:
Can you add some of your commit message (which is excellent) here? It would better define what is being tested.

It might also be useful to add somewhere that the root cause was that configuredServices were being considered when determining what masters provide what services, so a partially installed master could cause operations to fail on other masters, to the point where a broken master couldn't be removed.

Reply:
Added it to the docstring